### PR TITLE
Add fonts-conda-ecosystem as run_export

### PR DESCRIPTION
**Pull request by bgruening**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

This is in response to https://github.com/conda-forge/r-base-feedstock/issues/91

Basically, all R packages that need to plot something are failing in container environments as the fonts are missing.

### CODE REVIEW

**Comment by bgruening** [comment]
@conda-forge-admin, please rerender

**Comment by pkgw** [comment]
OK, just checking my understanding, mainly to help leave a documentation trail: with this change, packages that depend on `fontconfig` will also gain a runtime dep on `fonts-conda-ecosystem`, which ought to then make sure that base system fonts are available when such packages are installed ... correct?

**Comment by bgruening** [comment]
@pkgw yes this is correct.

**Review by pkgw — APPROVED** [review]
This seems reasonable to me, but this is a piece of the ecosystem with which I'm not very familiar, so I'd like to leave it for a day or two to give other folks a chance to chime in if they'd like.

**Comment by bgruening** [comment]
Thanks for the review @pkgw!

**Comment by bgruening** [comment]
Thanks, everyone!
